themes/agency/static/img/logo.svg

```diff
@@ -21,7 +27,7 @@
             <text x="1409.79px" y="209.155px" style="font-family:'Futura-Medium', 'Futura', sans-serif;font-weight:500;font-size:210.767px;fill:white;">A</text>
         </g>
         <g transform="matrix(0.410944,0,0,0.410944,992.799,166.714)">
-            <text x="978.64px" y="198.071px" style="font-family:'Futura-Medium', 'Futura', sans-serif;font-weight:500;font-size:210.767px;fill:white;">2<tspan x="1097.13px 1215.62px 1334.11px " y="198.071px 198.071px 198.071px ">019</tspan></text>
+            <text x="978.64px" y="198.071px" style="font-family:'Futura-Medium', 'Futura', sans-serif;font-weight:500;font-size:210.767px;fill:white;">2<tspan x="1097.13px 1215.62px 1334.11px " y="198.071px 198.071px 198.071px ">020</tspan></text>
```

Comment:
I could have done this

Reply:
Huh, the way I did it was way more complicated than this.